test/test_transformers.py

```diff
@@ -2572,13 +2576,13 @@ def test_mem_eff_backwards_determinism(self, device):
     @unittest.skipIf(not PLATFORM_SUPPORTS_MEM_EFF_ATTENTION, "Does not support SDPA")
     @unittest.skipIf(IS_JETSON, "causing sigkill on Jetson")
     @parametrize("batch_size", [1, 8])
-    @parametrize("seq_len_q", [4, 8, 64, 128, 256, 512, 1024, 2048] if SM80OrLater else [4, 8, 64, 128, 256, 512])
-    @parametrize("seq_len_k", [4, 8, 64, 128, 256, 512, 1024, 2048] if SM80OrLater else [4, 8, 64, 128, 256, 512])
-    @parametrize("head_dim", [8, 16, 32, 64, 72, 96, 128] if SM80OrLater else [8, 16, 32, 64])
+    @parametrize("seq_len_q", [4, 8, 64, 128, 256, 512, 1024, 2048] if SM80OrLater or TEST_WITH_ROCM else [4, 8, 64, 128, 256, 512])
```

Comment:
maybe we should extract this to something like, `platform supports long sequences`?

Reply:
Replaced with `MEM_EFF_CAPABILITY_MATCHES_SM80` in the latest commit